python/taichi/lang/__init__.py

```diff
@@ -481,15 +478,14 @@ def try_check_version():
                 with open(timestamp_path, 'w') as f:
```

Comment:
We may not need a timestamp file. Reading the mtime should work.

Reply:
What do you mean by mtime?

Reply:
last modified time returned by `os.stat(filename)`

Reply:
I see. However, I don't think we can guarantee that users won't modify the file. Maybe we should stay timestamp for now?

Reply:
Reasonable, but can we store timestamp with the version in the same file, maybe a json file for convenience. In this way we don't scatter misc files around.

Reply:
Understood, let me adjust this.